float32 leftAngle
float32 leftDist
float32 rightAngle
float32 rightDist
float32 frontDifference
float32 safeDirection
